x-pack/plugins/siem/public/components/page/hosts/authentications_table/index.tsx

```diff
@@ -162,7 +162,41 @@ const getAuthenticationColumns = (startDate: number): Array<Columns<Authenticati
     name: i18n.FAILURES,
     truncateText: false,
     hideForMobile: false,
-    render: ({ node }) => defaultToEmptyTag(node.failures),
+    render: ({ node }) => {
+      const id = escapeDataProviderId(
+        `authentications-table-${node._id}-node-failures-${node.failures}`
+      );
+      return (
+        <DraggableWrapper
+          key={id}
+          dataProvider={{
+            and: [],
+            enabled: true,
+            id,
+            name: 'authentication_failure',
+            excluded: false,
+            kqlQuery: '',
+            queryMatch: {
+              field: 'event.type',
+              value: 'authentication_failure',
```

Comment:
You'll want the actual value here, no? So `node.failures`? Also, is `event.type` the correct field?